src/main/java/org/broadinstitute/hellbender/tools/walkers/filters/VariantFiltration.java

```diff
@@ -50,13 +50,52 @@
  *   -R reference.fasta \
  *   -V input.vcf.gz \
  *   -O output.vcf.gz \
- *   --filter-expression "AB < 0.2 || MQ0 > 50" \
- *   --filter-name "my_filters"
+ *   --filter-name "my_filter1" \
+ *   --filter-expression "AB < 0.2" \
+ *   --filter-name "my_filter2" \
+ *   --filter-expression "MQ0 > 50"
  * </pre>
  *
  * <h3>Note</h3>
- * <p>Composing filtering expressions can range from very simple to extremely complicated depending on what you're
- * trying to do. Please see <a href='https://software.broadinstitute.org/gatk/documentation/article.php?id=1255'>this
+ * <p>
+ * Composing filtering expressions can range from very simple to extremely complicated depending on what you're
+ * trying to do.
+ * <p>
+ * Compound expressions (ones that specify multiple conditions connected by &&, AND, ||, or OR, and reference
+ * multiple attributes) require special consideration. By default, variants that are missing one or more of the
+ * attributes referenced in a compound expression are treated as PASS for the entire expression, even if the variant
+ * would satisfy the filter criteria for another part of the expression. This can lead to unexpected results if any
+ * of the attributes referenced in a compound expression are present for some variants, but missing for others.
+ * <p>
+ * It is strongly recommended that such expressions be provided as individual arguments, each referencing a
+ * single attribute and specifying a single criteria. This ensures that all of the individual expression are
+ * applied to each variant, even if a given variant is missing values for some of the expression conditions.
+ * <p>
+ * As an example, multiple individual expressions provided like this:
+ * <pre>
+ *   gatk VariantFiltration \
+ *   -R reference.fasta \
+ *   -V input.vcf.gz \
+ *   -O output.vcf.gz \
+ *   --filter-name "my_filter1" \
+ *   --filter-expression "AB < 0.2" \
+ *   --filter-name "my_filter2" \
+ *   --filter-expression "MQ0 > 50"
+ * </pre>
+ *
+ * <p>are preferable to a single compound expression such as this:
+ *
+ *  <pre>
+ *    gatk VariantFiltration \
+ *    -R reference.fasta \
+ *    -V input.vcf.gz \
+ *    -O output.vcf.gz \
+ *    --filter-name "my_filter" \
+ *    --filter-expression "AB < 0.2 || MQ0 > 50"
+ *  </pre>
+ *
+ * <p>
+ * </p>Please see <a href='https://software.broadinstitute.org/gatk/documentation/article.php?id=1255'>this
```

Comment:
This article is recommending the old way of filtering, are there plans to update it as part of this?